### ISSUE TITLE
Repeater + checkbox issue. Auto-checked after saving

### ISSUE THREAD

**Issue by AlxMedia** [themeum/kirki#1715]
### Issue description:
I have a checkbox inside of a repeater field. It is unchecked, or false, by default. If I add a few repeater items, each with a checkbox unchecked and then press Save, all the checkboxes are then true/checked. Not sure if I'm doing anything wrong here or if it's an issue. https://i.imgur.com/7j8EJHr.png (Open in new window)

```
'social-target'	=> array(
			'type'			=> 'checkbox',
			'label'			=> esc_attr__( 'Open in new window', 'flexwall' ),
			'description'	=> esc_attr__( '', 'flexwall' ),
			'default'		=> false,
		),
```

Another thing is that fontawesome fields does not work within the repeater.

**Comment by aristath** [comment]
fixed in the develop branch, will be included in v3.0.21 👍

**Comment by AlxMedia** [comment]
👍 That was quick.
